src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -351,6 +357,7 @@ public static PreferenceSettings Load(string filePath)
                 using (var fs = new FileStream(filePath, FileMode.Open, FileAccess.Read))
                 {
                     settings = serializer.Deserialize(fs) as PreferenceSettings;
+                    settings.InitializeNamespacesToExcludeFromLibrary();
```

Comment:
This `Load(string filePath)` method will not get called if `DynamoSettings.xml` does not previously exist in the APPDATA folder. Will discuss about this.